tests/util.py

```diff
@@ -122,6 +121,32 @@ def mkfs(device, fs="ext4"):
 
     subprocess.check_call(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, timeout=120)
 
+def xfs_repair_version():
+    cmd = ["xfs_repair", "-V"]
+    version = subprocess.check_output(cmd, timeout=10).rstrip().decode("utf-8").split(" ")[2]
+    return version
+
+def fsck(image, fs="ext4"):
+    if fs == 'xfs':
+        cmd = ["xfs_repair", "-n", "-v", image]
+    else:
+        cmd = ["fsck." + fs, "-n", "-f", "-v", image]
+
+    subprocess.check_call(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, timeout=10)
+
+def update_img(device, cow_file, bkp):
+    cmd = ["../utils/update-img", device, cow_file, bkp]
+    subprocess.check_call(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, timeout=180)
```

Comment:
Well, there is an easy way to warn user about library absence at least by removing `stderr=subprocess.DEVNULL`. In this case user will see an error
```
../utils/update-img: error while loading shared libraries: libelastio-snap.so.1: cannot open shared object file: No such file or directory
```
What do you think?

Reply:
I agree with that.